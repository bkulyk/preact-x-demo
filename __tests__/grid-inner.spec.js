import React from 'react';
import GridInner from '@bitchin/grid-inner';
import { shallow } from './enzyme';
import { it, expect } from './jest';

it('should render correctly', () => {
  expect(shallow(<GridInner>Something</GridInner>)).toMatchSnapshot();
});

it('should render correctly with props', () => {
  expect(shallow((
    <GridInner data-something="test" className="something">
      Something
    </GridInner>
  ))).toMatchSnapshot();
});
