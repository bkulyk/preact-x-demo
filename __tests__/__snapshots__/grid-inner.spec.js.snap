// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="mdc-layout-grid__inner"
>
  Something
</div>
`;

exports[`should render correctly with props 1`] = `
<div
  className="mdc-layout-grid__inner something"
  data-something="test"
>
  Something
</div>
`;
